server/services/utils/permissions/access/getOverviewAccessStatusCode.ts

```diff
@@ -25,5 +29,9 @@ export default function getOverviewAccessStatusCode(user: HmppsUser, prisoner: P
     return globalSearchUser || inactiveBookingsUser ? HmppsStatusCode.OK : HmppsStatusCode.PRISONER_IS_TRANSFERRING
   }
 
+  if (globalSearchUser && !options.allowGlobal) {
+    return HmppsStatusCode.GLOBAL_USER_NOT_PERMITTED
```

Comment:
What about users that have the right caseload and just happen to also have global search?

Maybe should change to `if (globalSearchUser && !options.allowGlobal && !inUsersCaseload) {`